src/setting_templates/ForagingSettings.json

```diff
@@ -18,5 +18,6 @@
     "go_cue_decibel_box1":73,
     "go_cue_decibel_box2":73,
     "go_cue_decibel_box3":73,
-    "go_cue_decibel_box4":73
+    "go_cue_decibel_box4":73,
+    "add_default_project_name":false
```

Comment:
I would set the default in the settings_template to make the default in the code (true, not false)

Reply:
changed